src/crypto/password/argon2.clj

```diff
@@ -0,0 +1,37 @@
+(ns crypto.password.argon2
+  "Functions for encrypting passwords using the recommended argon2 algorithm.
+
+  See:
+  https://infosecscout.com/best-algorithm-password-storage
+  https://github.com/phxql/argon2-jvm"
+  (:import (de.mkammerer.argon2 Argon2 Argon2Factory Argon2Advanced)))
```

Comment:
Can you change the `()` into `[]` to ensure it's consistent with the rest of the library?